reference/functions/pgroonga-condition.md

```diff
@@ -0,0 +1,112 @@
+---
+title: pgroonga_condition function
+upper_level: ../
+---
+
+# `pgroonga_condition()` function
+
+## Summary
+
+`pgroonga_condition()`関数は`pgroonga_condition`型の値を返します。
+関数名と型名が同じですが別物です。
+`pgroonga_condition`型は`pgroonga_full_text_search_condition`型や`pgroonga_full_text_search_condition_with_scorers`型のように複雑な条件式を表現します。
+
+`pgroonga_condition()`関数は`pgroonga_condition`型の値を作るための便利関数です。
+特定の属性値のみを指定して`pgroonga_condition`型の値を作れます。
+
+`pgroonga_full_text_search_condition`型や`pgroonga_full_text_search_condition_with_scorers`型では、このような便利関数がなかったため
+必ず全ての属性値を指定して値を作る必要がありました。
+
+したがって、不要な属性値があっても、`pgroonga_full_text_search_condition`型や`pgroonga_full_text_search_condition_with_scorers`型では、
+以下のように不要な属性値には`NULL`を指定する必要がありました。
+
```

Comment:
不要な属性値とは、「値を作るのに不要な属性値」であってます。

不要な属性値なんですが、属性値には何かしら値を入れないと行けないので、無効な値であることを表す`NULL`を入れる必要がある。ということを言いたいです。

Reply:
Therefore, you need to designate `NULL` for disused attribute value as follows when `pgroonga_full_text_search_condition` type and `pgroonga_full_text_search_condition_with_scorers` type are used to avoid making all the values.

それであれば不必要の　unnecessaryより、不使用的な意味にのdisusedのほうが良さそうなのでそちらに修正しました。

よろしくお願いいたします。